ext/standard/basic_functions.stub.php

```diff
@@ -113,33 +113,33 @@ function in_array(mixed $needle, array $haystack, bool $strict = false): bool {}
 function array_search(mixed $needle, array $haystack, bool $strict = false): int|string|false {}
 
 /** @prefer-ref $array */
-function extract(array &$array, int $extract_type = EXTR_OVERWRITE, string $prefix = ""): int {}
+function extract(array &$array, int $flags = EXTR_OVERWRITE, string $prefix = ""): int {}
 
 /**
  * @param array|string $var_name
  * @param array|string $var_names
  */
 function compact($var_name, ...$var_names): array {}
 
-function array_fill(int $start_key, int $count, mixed $value): array {}
+function array_fill(int $start, int $length, mixed $value): array {}
 
 function array_fill_keys(array $keys, mixed $value): array {}
 
 /**
- * @param string|int|float $low
- * @param string|int|float $high
+ * @param string|int|float $start
+ * @param string|int|float $end
  */
-function range($low, $high, int|float $step = 1): array {}
+function range($start, $end, int|float $step = 1): array {}
 
 function shuffle(array &$array): bool {}
 
-function array_pop(array &$stack): mixed {}
+function array_pop(array &$array): mixed {}
 
-function array_shift(array &$stack): mixed {}
+function array_shift(array &$array): mixed {}
 
-function array_unshift(array &$stack, mixed ...$values): int {}
+function array_unshift(array &$array, mixed ...$values): int {}
 
-function array_splice(array &$array, int $offset, ?int $length = null, mixed $replacements = []): array {}
+function array_splice(array &$array, int $offset, ?int $length = null, mixed $replacement = []): array {}
```

Comment:
After thinking about this a bit, I've normalized things to use `$offset` everywhere (in ext/standard -- I'll update ext/mbstring if we agree on this). Based on current use, `$offset` is more common (by 2-3 times) and a bit more flexible in usage (the strrpos issue mentioned). It's also already used in file-system related APIs, where it seems more natural (especially as in some APIs like fseek it can be an offset from different "whence").

Reply:
Yeah, seems a sensible idea!